### DISCUSSION TITLE
Train on Multiple loss functions one after other

### DISCUSSION THREAD

**Discussion by abhijeetdhakane** [Lightning-AI/pytorch-lightning#14728] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Is there any way, to train a network on multiple loss functions one after another,

I was doing something below in native pytorch:
```
mse_loss = nn.MSELoss()
l1_loss = nn.L1Loss()

for losses in [l1_loss, mse_loss, l1_loss, mse_loss]
    for epoch in range(3):
        data_vsl, data_txt = next(iter(train_loader))
        vsl_out = vsual_model(daat_vls)
        txt_out = text_model(data_txt)
        loss = losses(vsl_out, txt_out)

        loss.backward()
        optimizer.step()
```

Is there any way I can implement above steps in lightning?

**Comment by abhijeetdhakane** [comment]
Does it possible with `Loop`\ `FitLoop` or something similar like `optimizer_configurations` ?

**Comment by abhijeetdhakane** [comment]
#2791
